bundles/org.openhab.binding.openweathermap/src/main/resources/OH-INF/update/instructions.xml

```diff
@@ -143,6 +143,17 @@
 				<type>openweathermap:snow</type>
 			</add-channel>
 		</instruction-set>
+		<instruction-set targetVersion="2">
+			<add-channel id="moonrise" groupIds="forecastDaily">
+				<type>openweathermap:moonrise</type>
+			</add-channel>
+			<add-channel id="moonset" groupIds="forecastDaily">
+				<type>openweathermap:moonset</type>
+			</add-channel>
+			<add-channel id="moon-phase" groupIds="forecastDaily">
+				<type>openweathermap:moon-phase</type>
+			</add-channel>
```

Comment:
I think group id `forecastToday` should be added since the channels were also added to channel group type id `oneCallDaily` referenced by that group?
```suggestion
			<add-channel id="moonrise" groupIds="forecastToday,forecastDaily">
				<type>openweathermap:moonrise</type>
			</add-channel>
			<add-channel id="moonset" groupIds="forecastToday,forecastDaily">
				<type>openweathermap:moonset</type>
			</add-channel>
			<add-channel id="moon-phase" groupIds="forecastToday,forecastDaily">
				<type>openweathermap:moon-phase</type>
			</add-channel>
```

Reply:
Yes. Shouldn't those also be added to `forecastTomorrow`, `forecastDay2` ... `forecastDay7`?

Reply:
Oh, right, I missed those. 🙄

Reply:
I will add them, I've already helped with this PR.

Reply:
Done :+1: